docs/src/main/tut/json.md

```diff
@@ -31,9 +31,9 @@ community.  The functionality is similar:
 
 ```scala
 libraryDependencies += Seq(
-  "org.http4s" %% "http4s-argonaut" % "{{< version >}}",
+  "org.http4s" %% "http4s-argonaut" % "{{< http4sVersion >}}",
   // Optional for auto-derivation of JSON codecs
-  "com.github.alexarchambault" %% "argonaut-shapeless_6.2" % "1.2.0-M4"
+  "com.github.alexarchambault" %% "argonaut-shapeless_6.2" % "1.2.0-M5"
```

Comment:
Skipped it because `argonaut-shapeless_6.2` is not a dependency of the `docs` module, so it is not used when compiling the tuts.  Felt more arbitrary because there's no compiler dependency to fall out-of-sync with.  Happy to add it if you think it's worthwhile.